study.md

```diff
@@ -95,11 +104,11 @@ would use to satisfy these requirements.
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+Application seed data allows you to add inital data after a database is created.
 ```
 
 Should you use seeds to create data to experiment with during development?
 
 ```md
-<!-- your response here -->
+Yes you should.
 ```
```

Comment:
Depending on who you ask. I don't want to say flat out no, but it's considered better to populate you db with dummy data. I know I said I do it, but Jeff is a more experience developer than I and the way mentioned is a better path.